### DISCUSSION TITLE
Timeline for release of Starlette Middleware

### DISCUSSION THREAD

**Discussion by danut-t** [p1c2u/openapi-core#726] [General]
Hey everyone! We really like this library and want to use it for our FastAPI app, using the Starlette Middleware we saw in the documentation. However, it looks like it hasn't been released yet (at least we couldn't find it in the 0.18.2 version that we've used). Is there an estimated release date for a new version that would include it? Many thanks in advance and for the work you're putting on this project!

**Comment by p1c2u** [comment]
Hi @danut-t 

the preview release of 0.19.x is coming soon, possibly this month. It needs one more change (including the middleware) to be finished.

As a side note, I have a plan to make integration with FastAPI itself. It will be possible after breaking changes made with 0.19.x

**Reply by danut-t** [reply]
Awesome, looking forward to both of them!
